src/applications/edu-benefits/10282/config/form.js

```diff
@@ -1,144 +1,195 @@
   },
+  transformForSubmit: transform,
   chapters: {
-    applicantInformationChapter: {
-      title: 'Applicant Information',
+    personalInformation: {
+      title: 'Your personal information',
       pages: {
-        applicantInformation: {
-          path: 'applicant-information',
-          title: 'Applicant Information',
+        applicantName: applicantInformationName.applicantInformationField(),
+        veteranDesc: {
+          title: 'Your personal information',
+          path: 'applicant-information-1',
+          uiSchema: veteranDesc.uiSchema,
+          schema: veteranDesc.schema,
+        },
+        contactInfo: {
+          title: 'Your personal information',
+          path: 'applicant-information-2',
+          uiSchema: applicantContactInfo.uiSchema,
+          schema: applicantContactInfo.schema,
+        },
+        applicantCountry: {
+          title: 'Your personal information',
+          path: 'applicant-information-3',
+          uiSchema: applicantInformationCountry.uiSchema,
+          schema: applicantInformationCountry.schema,
+        },
+        applicantState: {
+          title: 'Your personal information',
+          path: 'applicant-information-4',
+          uiSchema: applicantState.uiSchema,
+          schema: applicantState.schema,
+          depends: formData => {
+            return formData.country === 'United States';
+          },
+        },
+        genderRaceQuestion: {
+          title: 'Your personal information',
+          path: 'applicant-information-5',
+          uiSchema: genderRaceQuestion.uiSchema,
+          schema: genderRaceQuestion.schema,
+        },
+        applicantRaceAndEthnicity: {
+          title: 'Your personal information',
+          path: 'applicant-information-6',
+          uiSchema: applicantRaceAndEthnicity.uiSchema,
+          schema: applicantRaceAndEthnicity.schema,
+          depends: formData => formData.raceAndGender === 'Yes',
+        },
+        applicantGender: {
+          title: 'Your personal information',
+          path: 'applicant-information-7',
+          uiSchema: applicantGender.uiSchema,
+          schema: applicantGender.schema,
+          depends: formData => formData.raceAndGender === 'Yes',
+        },
+      },
+    },
+    educationAndEmploymentHistory: {
+      title: ' Your education and employment history',
+      pages: {
+        highestLevelOfEducation: {
+          title: 'Your education and employment history',
+          path: 'applicant-information-8',
+          uiSchema: highestLevelOfEducation.uiSchema,
+          schema: highestLevelOfEducation.schema,
+        },
+        currentlyEmployed: {
+          title: 'Your education and employment history',
+          path: 'applicant-information-9',
           uiSchema: {
-            fullName: fullNameUI,
-            ssn: ssnUI,
+            currentlyEmployed: {
+              'ui:title': (
+                <h3 className="vads-u-margin--0">Are you currently employed</h3>
```

Comment:
Should this have a question mark on it?

Reply:
I Think so but most of those did copy and baste from Jira lol

Reply:
done